samples/Blazor6/Server/Program.cs

```diff
@@ -38,6 +38,9 @@
         .AddCookie("cookie", options =>
         {
             options.Cookie.Name = "__Host-blazor";
+
+            // If your authority is on a different domain from your SPA (blazor in this case), you might want to use Lax instead of Strict
+            // See https://github.com/DuendeSoftware/Support/issues/565
```

Comment:
I'm not sure if this comment is correct. This sample is already working with "demo.duendesoftware.com" which is cross site from localhost, no?

Reply:
I think this only would be necessary if someone was using the custom middleware that issues a challenge that @maldworth added to their fork of the sample. Since that middleware isn't in the sample, I think we won't merge change, but thanks for the effort though! It was really interesting looking into this.